packages/config-utils/src/feo/spec/frontend-crd.schema.json

```diff
@@ -32,16 +32,44 @@
       ],
       "additionalProperties": false
     },
+    "apiSpecInfo": {
```

Comment:
Chatted with Justin offline - I'd like to make sure our AJV validation in frontend components mirrors what FEO expects (FEO is enforcing that either `versions` or `specs` is defined when specifying an `API` clause, otherwise an error is thrown). Can we try `anyOf` here?